.NET 5.0 platform

BeatSkip opened the pull request:
- Added conditional SDK selection
	(As .NET 5 uses 'Microsoft.NET.Sdk' instead
	of 'Microsoft.NET.Sdk.WindowsDesktop')
- Added .NET 5.0 Package includes

currently validating everything
but yet to see any problems

Roemer commented:
Awesome thank you. I will also do some checks soon so we can merge this change and do a new release.

BeatSkip commented:
i updated the versioning and cleaned the packagereferences to the newer format that you already used for the rest. will run functional tests today in a personal project completely built on .net 5.0

sharpordie commented:
Do you think we can soon use FlaUI with .NET5?

BeatSkip commented:
I am actually using it on .net 5.0 currently. The project isn't finalized yet, but everything seems to work just fine @sharpordie

Roemer commented on src/FlaUI.Core/FlaUI.Core.csproj:
```diff
@@ -1,15 +1,17 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
-
+﻿<Project>
+  <Sdk Condition="'$(TargetFramework)' != 'net5.0-windows'" Name="Microsoft.NET.Sdk.WindowsDesktop" />
```
Unfortunately this does not work as this adds the imports before everything else. Fortunately it is allowed to use `Microsoft.NET.Sdk.WindowsDesktop` also in .net 5 and there is no warning shown anymore. See https://github.com/dotnet/sdk/pull/13441

Roemer commented on src/FlaUI.Core/FlaUI.Core.csproj:
```diff
@@ -33,6 +35,11 @@
     <IntermediateOutputPath>obj\$(Configuration)\Signed</IntermediateOutputPath>
   </PropertyGroup>
 
+  <PropertyGroup>
```
Why was this added?

Roemer commented on src/FlaUI.UIA2/FlaUI.UIA2.csproj:
```diff
@@ -37,7 +39,7 @@
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' != 'netcoreapp3.0' Or '$(TargetFramework)' != 'netcoreapp3.1'">
+  <ItemGroup Condition="'$(TargetFramework)' != 'netcoreapp3.0' And '$(TargetFramework)' != 'netcoreapp3.1'">
```
Good catch ;) That removes some build warnings

Roemer commented:
I made a new commit on your behalf which should work fine: https://github.com/FlaUI/FlaUI/commit/8d736cbd36bb6b6e1957ee7765f3ab852c520326
So this PR should be obsolete. Thanks for your help!
